types/fuzzyset/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for fuzzset 1.0
+// Project: https://github.com/axiak/fuzzyset
+// Definitions by: Louis Grignon <https://github.com/lgrignon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface FuzzySet {
+    get(candidate: string): Array<[number, string]>;
+    add(value: string): boolean;
+    length(): number;
+    isEmpty(): boolean;
+    values(): string[];
+}
+
+declare function FuzzySet(source: string[], useLevenshtein?: boolean, gramSizeLower?: number, gramSizeUpper?: number): FuzzySet;
```

Comment:
Yes you are right, that's the syntax I use in my project, I recently added the module way so I join it to this PR (and add import in tests as well).
Still, the global way is also possible, both are coexistent in this library.